.github/workflows/docs.yml

```diff
@@ -0,0 +1,24 @@
+name: Deploy docs
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    container:
+      image: crystallang/crystal:0.36.1
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+      - name: "Install shards"
+        run: shards install
+      - name: "Generate docs"
+        run: crystal docs
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
did you already set it?

Reply:
Nope. This would actually be set under settings, but I don't have access to the settings on this repo.

The strange thing though is, we use this same template in Lucky, for example with Carbon https://luckyframework.github.io/carbon/ but we don't have this set anywhere... 

<img width="934" alt="Screen Shot 2021-06-16 at 10 03 57 AM" src="https://user-images.githubusercontent.com/2391/122262376-34b4b980-ce8a-11eb-8c32-793222f07a3e.png">


So I have no clue how it's working, but it does 😅

Reply:
hummm interesting hahahha so let's merge and see if it's gonna work :P